File: pinyougou-web-manager/src/main/webapp/js/service/userService.js
//服务层
app.service('userService',function($http) {
    //分页查询所有用户
    this.findUser = function () {
        return $http.get('../user/findUser.do');
    }
    //冻结用户 无法登陆 下单
    this.freeze = function(ids){
        return $http.get("../user/freeze.do?ids="+ids);
    }

    this.search= function(page,rows,searchEntity){
        return $http.post("../user/search.do?pageNum="+page+"&pageSize="+rows,searchEntity);
    }

});

File: pinyougou-interface/src/main/java/cn/itcast/core/service/UserService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.user.User;
import entity.PageResult;

import java.util.List;

public interface UserService {
    void sendCode(String phone);

    void add(User user, String smscode);

    List<User> findUser();

    PageResult search(Integer page, Integer rows, User user);

    Integer showUserCount();

    void freeze(Long[] ids);
}
